opt/cs50/bin/flask

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+# If $FLASK_APP isn't set, check for application.py (for backwards compatibility), else default to app.py
+if [[ -z "$FLASK_APP" ]]; then
+    if [[ -f "application.py" ]]; then
+        FLASK_APP="application.py"
+    else
+        FLASK_APP="app.py"
```

Comment:
This doesn't need to be set since it's the default?

Reply:
Good call.